index.js

```diff
@@ -72,7 +78,8 @@ function fastifyCors (fastify, opts, next) {
   // remove most headers (such as the Authentication header).
   //
   // This route simply enables fastify to accept preflight requests.
-  fastify.options('*', { schema: { hide: hideOptionsRoute } }, (req, reply) => {
+
+  fastify.options('*', { schema: { hide: hideOptionsRoute }, ...(routeLogLevel !== null && { logLevel: routeLogLevel }) }, (req, reply) => {
```

Comment:
```suggestion
  fastify.options('*', { schema: { hide: hideOptionsRoute }, routeOptions) }, (req, reply) => {
```

Maybe we should prepare a `routeOption` (or more explicit name) object with default to `{}`.